db/migrate/20150410025815_add_is_referral_to_group.rb

```diff
@@ -0,0 +1,9 @@
+class AddIsReferralToGroup < ActiveRecord::Migration
+  def change
+    add_column :groups, :is_referral, :boolean
+  end
+
+  Group.parents_only.where("created_at < '2015-04-9 1:00:00'").each do |g|
```

Comment:
Group.parents_only.where("created_at < '2015-04-9 1:00:00'").where(group_request_id: nil).update_all(is_referral: true)

This will do all groups in one query on the db, so no long runnning migrations. Please test.